packages/core/admin/admin/src/content-manager/history/components/VersionHeader.tsx

```diff
@@ -2,12 +2,14 @@ import * as React from 'react';
 
 import { HeaderLayout, Typography } from '@strapi/design-system';
 import { Link } from '@strapi/design-system/v2';
-import { useQueryParams } from '@strapi/helper-plugin';
+import { useQueryParams, LoadingIndicatorPage } from '@strapi/helper-plugin';
```

Comment:
This is a type right?
```suggestion
import { useQueryParams, type LoadingIndicatorPage } from '@strapi/helper-plugin';
```

Reply:
oh whoops I thought I saw LoadingIndicatorProps 🤦‍♂️